completion/available/makefile.completion.bash

```diff
@@ -1,3 +1,36 @@
-# Add completion for Makefile
-# see http://stackoverflow.com/a/38415982/1472048
-complete -W "\`grep -oE '^[a-zA-Z0-9_-]+:([^=]|$)' Makefile | sed 's/[^a-zA-Z0-9_-]*$//'\`" make
+#!/usr/bin/env bash
```

Comment:
I believe per this conversation https://github.com/Bash-it/bash-it/issues/1491#issuecomment-586882258 we don't need to include the bash shebang in sourced files.

Reply:
I was thinking the same, but saw it in some of the other completions. I'll remove it.